sdk/communication/communication-common/review/communication-common.api.md

```diff
@@ -110,34 +115,34 @@ export interface PhoneNumberKind extends PhoneNumberIdentifier {
     kind: "phoneNumber";
 }
 
-// @internal
-export const _serializeCommunicationIdentifier: (identifier: CommunicationIdentifier) => _SerializedCommunicationIdentifier;
+// @public
```

Comment:
I thought these are supposed to be hidden

Reply:
They are. But api-extractor doesn't understand @hidden.

Developers can still use those APIs, it's just a convention that they shouldn't and we won't generate docs for them on docs.microsoft.com.

See the linked issue #3448